lib/tapioca/compilers/dsl/active_storage.rb

```diff
@@ -0,0 +1,73 @@
+# typed: strict
+# frozen_string_literal: true
+
+begin
+  require "active_record"
+  require "active_storage"
+  require "active_storage/reflection"
+rescue LoadError
+  return
+end
+
+module Tapioca
+  module Compilers
+    module Dsl
+      # `Tapioca::Compilers::Dsl::ActiveStorage` decorates RBI files for subclasses of
+      # `ActiveRecord::Base` that declare [one](https://edgeguides.rubyonrails.org/active_storage_overview.html#has-one-attached)
+      # or [many](https://edgeguides.rubyonrails.org/active_storage_overview.html#has-many-attached) attachments.
+      #
+      # For example, with the following `ActiveRecord::Base` subclass:
+      #
+      # ~~~rb
+      # class Post < ApplicationRecord
+      #  has_one_attached :photo
+      #  has_many_attached :blogs
+      # end
+      # ~~~
+      #
+      # this generator will produce the RBI file `post.rbi` with the following content:
+      #
+      # ~~~rbi
+      # # typed: true
+      # class Post
+      #  def photo; end
+      #  def photo=(attachable); end
+      #  def blogs; end
+      #  def blogs=(attachable); end
+      # end
+      # ~~~
+      class ActiveStorage < Base
+        extend T::Sig
+
+        sig do
+          override.params(root: RBI::Tree,
+            constant: T.all(Module, ::ActiveStorage::Reflection::ActiveRecordExtensions::ClassMethods)).void
+        end
+        def decorate(root, constant)
+          return if constant.reflect_on_all_attachments.empty?
+
+          root.create_path(constant) do |scope|
+            constant.reflect_on_all_attachments.each do |reflection|
+              type = "T.untyped"
+              name = reflection.name.to_s
+
+              getter_sig = RBI::Sig.new(return_type: type)
+              setter_sig = RBI::Sig.new(params: [RBI::SigParam.new("attachable", type)])
+
+              scope << RBI::Method.new(name, sigs: [getter_sig])
+              scope << RBI::Method.new("#{name}=", params: [RBI::Param.new("attachable")],
```

Comment:
I think we prefer the utility methods instead of these low level ones:
```suggestion
			  scope.create_method(
			  	name, 
			  	return_type: type
			  ) 

        	  scope.create_method(
			  	"#{name}=", 
			  	parameters: [create_param("attachable", type: "T.untyped")],
			  	return_type: "T.untyped"
			  ) 
```